conf_files/pyro_camera.yaml

```diff
@@ -0,0 +1,99 @@
+# Config file for a distributed camera & focuser.
+#
+# Contains a default configuration with a simulated camera and focuser for testing
+# and example configurations for other camera and focusers (commented out).
+#
+# Do not edit this file, instead copy one of the examples to pyto_camera_local.yaml
```

Comment:
Is this supposed to be pyro_camera_local (not pyto_...)?

Reply:
Yep!